frontend/src/employee-frontend/components/child-information/DailyServiceTimesSection.tsx

```diff
@@ -195,6 +197,11 @@ const DailyServiceTimesSection = React.memo(function DailyServiceTimesSection({
           end:
             formData.type === 'REGULAR' ? required(formData.regular.end) : null
         },
+        irregular:
+          formData.type === 'IRREGULAR' &&
+          !weekdays.some((day) => formData[day].selected)
+            ? 'Valitse vähintään yksi päivä'
```

Comment:
could be moved into localization file

Reply:
The error message was not actually used anywhere, replaced with `true`